ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,338 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+from urlparse import urlparse, urlunsplit
+import rdflib
+
+
+class G2PDataset:
+    """
+    an rdf object store
```

Comment:
We need better documentation here. What are we doing? We're translating data from (somewhere) in (some format) into GA4GH values, right? This should be spelled out here.